src/services/messagesService.ts

```diff
@@ -192,7 +193,7 @@ const postRichObjectToConversation = async function(token: string, { objectType,
  * @param lastReadMessage id of the last read message to set
  * @param options request options
  */
-const updateLastReadMessage = async function(token: string, lastReadMessage: setReadMarkerParams, options: object): setReadMarkerResponse {
+const updateLastReadMessage = async function(token: string, lastReadMessage?: number|null, options?: object): setReadMarkerResponse {
```

Comment:
```suggestion
const updateLastReadMessage = async function(token: string, lastReadMessage?: number|null|undefined, options?: object): setReadMarkerResponse {
```

?

Reply:
`lastReadMessage?:` is already undefined

Reply:
Ah okay, I thought it's better to specify it too in the possible types